app/component/route/route-header.cjsx

```diff
@@ -2,24 +2,25 @@ React                 = require 'react'
 Icon                  = require '../icon/icon'
 GtfsUtils             = require '../../util/gtfs'
 Link                  = require 'react-router/lib/Link'
-
+classnames            = require 'classnames'
 
 class RouteHeader extends React.Component
```

Comment:
we could use the PropTypes on every component both documentation (which props are injected to the component) and also to ensure that the type is correct of the props.
https://facebook.github.io/react/docs/reusable-components.html